scripts/delete-workflow-runs.ps1

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env pwsh
```

Comment:
Can you add execute permission on the script?

Reply:
done